test/e2e/playwright/consolidation-orders.spec.ts

```diff
@@ -29,9 +29,12 @@ test.describe('Consolidation Orders', () => {
     page,
   }) => {
     // get pending consolidation order id
-    const pendingConsolidationOrder: Order = orderResponseBody.find(
-      (o) => o.orderType === 'consolidation' && o.status === 'pending',
-    );
+    const pendingConsolidationOrder = orderResponseBody
```

Comment:
I think this should be reverted. The e2e test fails locally if we have consolidated cases in the first accordion. Workaround is we need to reload orders before running the e2e test.